packages/component/src/Activity/Speak.js

```diff
@@ -58,14 +61,9 @@ Speak.propTypes = {
     text: PropTypes.string
   }).isRequired,
   markAsSpoken: PropTypes.func.isRequired,
-  selectVoice: PropTypes.func.isRequired,
-  styleSet: PropTypes.shape({
-    options: PropTypes.shape({
-      showSpokenText: PropTypes.bool.isRequired
-    }).isRequired
-  }).isRequired
+  selectVoice: PropTypes.func.isRequired
 };
 
-export default connectSpeakActivity(({ styleSet }) => ({ styleSet }))(Speak);
+export default connectSpeakActivity()(Speak);
```

Comment:
Same comment as above - we can get rid of `connectSpeakActivity` and use hooks instead or update `connectSpeakActivity`  to use hooks instead of `connectToWebChat`